CONTRIBUTING.md

```diff
@@ -153,6 +153,18 @@ get it merged. You can also post in the `#docs` [Slack
 channel](https://knative.slack.com/) to get input on your ideas or find areas to
 contribute before creating a PR.
 
+### Common Troubleshooting issues for PRs
+
+1. CLA fails even though you have signed a CLA. This can happen if someone accepts and commits suggestions due to the email address not matching the address on record for the CLA. 
+
+1. Test Fails. If you do not see a specific error related to a change you made, and instead the errors are related to timeouts then you may want to rerun the test at a later time. There are running tasks that could result in timeouts or rate limiting if your test runs at the same time as the tasks. 
+
+1. Previews are not working - WIP (not sure what's causing this)
```

Comment:
This looks like stagebot may be fixed now @mattmoor ? It has been working for me lately.

Reply:
It only runs when the `/approve` bit is flipped, which makes it seem somewhat unreliable.

This is basically the only piece of knobots that IDK how to turn into github actions, and so I've kinda been hoping a good alternative would present itself.  Perhaps we could even get proper netlify preview on this repo (like on website)? @evankanderson @RichieEscarez would know more.